### ISSUE TITLE
Create Initial User Guide

### ISSUE THREAD

**Issue by oleg-nenashev** [gradle/declarative-gradle#65]
#63 provides only the initial Getting started material but not a continuous user guide. It would be nice to create some entries for it too.

In particular, I would explicitly set expectation about regular updates while in Alpha, as @tresat suggested in #63 :

> 🤔 I would make this disclaimer even more prominent. I try to keep the wrapper up-to-date, but sometimes miss it or it isn't immediately possible (waiting on reviews). I'd make this a callout and note that running this project frequently requires bleeding edge features and experiments only available in the nightly, and that you might have to manually download it, etc.

**Comment by eskatos** [comment]
* Superseded by https://github.com/gradle/gradle/issues/36371
